### ISSUE TITLE
Cert checking is overly restrictive

### ISSUE THREAD

**Issue by kilasuit** [PowerShell/PowerShellGetv2#579]
In some of our internal modules we are using Lability https://github.com/VirtualEngine/Lability which recently changed a few minor details in their code signing cert which has caused PowerShellGet to throw an error as per https://github.com/VirtualEngine/Lability/issues/382

This effects v2.2.3 of PowerShellGet

Whilst the scenario to change locations isn't likely very frequent, it is still valid and should not throw an error like this and allow without the need of `SkipPublisherCheck` parameter 

![image](https://user-images.githubusercontent.com/6355225/73747589-a38aea00-474f-11ea-9a0e-eb253a5ba4d6.png)

Note that in the above only the C fields in the cert have changed - the O hasn't and that's really what we should be matching on within checks in PowerShellGet

**Comment by Shane-OH** [comment]
Having the same problem even with PSGet v 2.2.5, did you ever manage to get a fix for this? Passing the -SkipPublisherCheck param isnt an option as I'm relying on RequireModules in the psd to install the module.

![image](https://user-images.githubusercontent.com/74359284/98940634-2f058880-24e3-11eb-93fb-ee04cb32d9b5.png)

**Comment by kduvzc** [comment]
Yeah... as @Shane-OH reported this is not allowing PowershellGet to install AWSPowerShell.NetCore 4.1.2.0 on a machine which already has AWSPowershell.NetCore 4.0.5.0 already installed because the root certification authorities differ: AWSPowershell.NetCore 4.0.5.0's root certification authority is **VeriSign Universal Root Certification Authority** while version 4.1.2.0 has **Amazon Web Services, Inc.** .
